addon/components/ember-table/template.hbs

```diff
@@ -1,8 +1,15 @@
-<table>
-  {{yield (hash
-    api=api
-    head=(component "ember-thead" api=api)
-    body=(component "ember-tbody" api=api)
-    foot=(component "ember-tfoot" api=api)
-  )}}
-</table>
+<div
```

Comment:
this is probably a semver breaking change as it alters the DOM?

Reply:
Yes, that thought crossed my mind, but positioning the scroll indicators on top of the table requires a common wrapping element 😬

Reply:
Retargeted to `3.0-beta` branch which should make this breaking change more palatable